### ISSUE TITLE
Unused false positives when used in script tag attribute value

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#593]
This code is incorrectly triggering UNUSED_METHOD_ARGUMENT and UNUSED_LOCAL_VARIABLE even though they are being used in `cfhttpparam` script tags. I have verified that this works fine for the standard tag equivalent.

```
/**
 * Some function hint
 * @call The value of call
 */
public void function foo(
	required string call
)
{
	var bar = "";
	cfhttp(url="#variables.host#", port="22", method="GET", timeout="30", result="local.httpResponse") {
		cfhttpparam(type="url", name="call", value="#arguments.call#");
		cfhttpparam(type="url", name="bar", value="#local.bar#");
	}
}
```

- Argument  call is not used in function. Consider removing it. (UNUSED_METHOD_ARGUMENT)
- Local variable bar is not used in function foo. Consider removing it. (UNUSED_LOCAL_VARIABLE)

Related: #574

**Comment by ryaneberly** [comment]
cfparser does not include the body of cfhttp in the parsed tree.

**Comment by ryaneberly** [comment]
fixed in dev branch

**Comment by ryaneberly** [comment]
fixed in dev branch
